services/src/main/java/io/grpc/protobuf/services/ChannelzProtoUtil.java

```diff
@@ -468,4 +492,11 @@ private static <T> T getFuture(ListenableFuture<T> future) {
       throw Status.INTERNAL.withCause(e).asRuntimeException();
     }
   }
+
+  private static Any packToAny(String typeName, MessageLite value) {
```

Comment:
Should we have a comment "somewhere" saying this is trying to work with javalite?

Reply:
Sure. Added a comment here and in the class javadoc.